cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -398,6 +392,22 @@ pub enum VecRRNarrowOp {
     Uqxtn64,
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum VecRRRLongOp {
+    /// Signed multiply long.
+    Smull8,
+    Smull16,
+    Smull32,
+    /// Unsigned multiply long.
+    Umull8,
+    Umull16,
+    Umull32,
+    /// Unsigned multiply add long
+    Umlal8,
+    Umlal16,
```

Comment:
It seems like this and `Umlal8` aren't actually generated in any lowerings (unless I'm missing something), but do you want to leave these in for completeness with possible future lowerings?

Reply:
Yes, exactly - thought it would be weird to just port the 32-bit version.